shared/src/business/entities/judgeActivityReport/JudgeActivityReportSearch.ts

```diff
@@ -83,12 +85,18 @@ export class JudgeActivityReportSearch extends JoiValidationEntity {
       .array()
       .items(joi.string())
       .description('The last names of judges to generate report for'),
-    pageSize: joi
+    pageNumber: joi
       .number()
       .optional()
       .description(
         'The page size for getting aggregated judge activity report for closed cases, opinions, orders issued  and sessions held',
       ),
+    pageSize: joi
+      .number()
+      .optional()
+      .description(
+        'The page number for calculating the number of cav and submitted cases to display',
```

Comment:
Nit

Reply:
Will be fixed on the integration test branch being worked on now.